script.js

```diff
@@ -170,14 +171,47 @@ function closePopup() {
   const unpopElement = document.getElementById('desk-popup');
   unpopElement.parentNode.removeChild(unpopElement);
 }
+
+// Local Storage
 const contactForm = document.getElementById('contact-form-validate');
 contactForm.addEventListener('submit', (event) => {
   event.preventDefault();
   document.getElementById('error-msg').innerText = '';
   const email = document.getElementById('form-email').value;
+  const formData = {
+    name: document.getElementById("fullname").value,
+    email: email,
+    message: document.getElementById("text-area").value,
+  }
+  localStorage.setItem("key",JSON.stringify(formData))
+  
   if (email !== email.toLowerCase()) {
     document.getElementById('error-msg').innerText = 'Email Address should be in Lower case';
   } else {
     document.forms[0].submit();
+    localStorage.clear();
   }
```

Comment:
- [ ] Form values are not stored in `localStorage` as required by the project milestone. Please find out why and fix the bug.

> You can use the guide below 👇🏻

- The form data for all inputs should be stored in a single object, saved to a single local storage registry.
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```
- use the `localStorage.getItem` and `localStorage.setItem` functions to save and retrieve the form data object to the local storage.

```
localStorage.setItem('contactFormInput', JSON.stringify(formData));

let savedFormData = localStorage.getItem('contactFormInput');
savedFormData = JSON.parse(savedFormData);

```